grails-plugin-gsp/src/test/groovy/org/grails/plugins/web/DefaultDateHelperSpec.groovy

```diff
@@ -217,7 +217,7 @@ class DefaultDateHelperSpec extends Specification {
 
         given:
         DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd").withZone(ZoneId.of('UTC'))
-        def date = new java.sql.Date(localDate.atStartOfDay(ZoneId.of("UTC")).toInstant().toEpochMilli())
+        def date = new java.sql.Date(localDate.atStartOfDay(ZoneId.systemDefault()).toInstant().toEpochMilli())
```

Comment:
why did you go from UTC to systemDefault() ?

Reply:
The test failed with it as UTC.

Reply:
'1941-01-04' != '1941-01-05'

Reply:
It would pass when run on any machine in UTC or UTC+X, but fail for any in UTC-X.  This change should allow it to pass regardless of system ZoneId.